Keyboard-Kit.podspec

```
@@ -0,0 +1,32 @@
+Pod::Spec.new do |spec|
+    spec.name = 'Keyboard-Kit'
+    spec.module_name = 'KeyboardKit'
+    spec.version = '0.0.1'
+    spec.license = { :type => 'MIT', :file => 'License.txt' }
+    spec.homepage = 'https://github.com/douglashill/KeyboardKit'
+    spec.authors = { 'Douglas Hill' => 'https://twitter.com/qdoug' }
+    spec.summary = 'A framework to help iOS and Mac Catalyst apps support being controlled using a hardware keyboard.'
+
+    spec.description = <<-DESC
+KeyboardKit is a framework to help iOS and Mac Catalyst apps support being controlled using a hardware keyboard.
+
+Keyboard control is a standard expectation of Mac apps. It’s important on iOS too because a hardware keyboard improves speed and ergonomics, which makes an iPad an even more powerful productivity machine.
+
+Apps created with AppKit tend to have better support for keyboard control compared to UIKit-based apps. I believe the principal reason for this is that most AppKit components respond to key input out of the box, while most UIKit components do not. KeyboardKit aims to narrow this gap by providing subclasses of UIKit components that respond to key commands.
+                       DESC
+
+    spec.source = { :git => 'https://github.com/douglashill/KeyboardKit.git', :tag => spec.version.to_s }
+    spec.swift_version = '5.0'
+    spec.ios.deployment_target  = '13.0'
+    spec.source_files = 'KeyboardKit/*.{h,m,swift}'
+    spec.public_header_files = [
+      'KeyboardKit/KeyboardKit.h',
+      'KeyboardKit/BarButtonItem.h',
+      'KeyboardKit/KeyValueCoding.h'
+    ]
+    spec.exclude_files = [
+      'KeyboardKit/info.plist',
```

I guess this needs to match the capitalisation?

```suggestion
      'KeyboardKit/Info.plist',
```

---

Everything was working as expect but nicely spotted 🏅